test/jdk/java/net/httpclient/AsFileDownloadTest.java

```diff
@@ -407,6 +448,21 @@ public void handle(Http2TestExchange t) throws IOException {
                 os.write(bytes);
             }
         }
+
+        @Override
+        public void handle(HttpTestExchange t) throws IOException {
```

Comment:
this is almost an exact copy of the method above.

Reply:
Right. Thanks for spotting that. The right thing to do here would be to refactor the HTTP/2 part of the test to stop using Http2TestServer but use HttpTestServer instead, like the new HTTP/3 part. That would get rid of the almost duplicate. I have logged https://bugs.openjdk.org/browse/JDK-8365802